docs-ref-conceptual/includes/cli-install-linux-apt.md

```diff
@@ -40,25 +40,26 @@ If you prefer a step-by-step installation process, complete the following steps
 
     ```bash
     sudo apt-get update
-    sudo apt-get install ca-certificates curl apt-transport-https lsb-release gnupg
+    sudo apt-get install apt-transport-https ca-certificates curl gnupg2 lsb-release
     ```
 
 2. Download and install the Microsoft signing key:
 
     ```bash
-    sudo mkdir -p /etc/apt/keyrings
-    curl -sLS https://packages.microsoft.com/keys/microsoft.asc |
-        gpg --dearmor |
-        sudo tee /etc/apt/keyrings/microsoft.gpg > /dev/null
-    sudo chmod go+r /etc/apt/keyrings/microsoft.gpg
+    curl -fssL https://packages.microsoft.com/keys/microsoft.asc |\
```

Comment:
```suggestion
    sudo mkdir -p /etc/apt/keyrings
    curl -fssL https://packages.microsoft.com/keys/microsoft.asc |\
```
`/etc/apt/keyrings` does not exists in Ubuntu 20.04 and 18.04.

Reply:
indeed it should be